en/appendices/4-0-migration-guide.rst

```diff
@@ -452,6 +464,11 @@ ORM
   the ``TypeFactory`` as the default ``datetime`` type or re-mapping individual
   columns. See the Database migration notes for how this type is automatically
   mapped to database types.
+* A new type class ``DateTimeTimezoneType`` has been added for datetime types
+  with time zone support. You can opt into using this type by adding it to
+  the ``TypeFactory`` as the default ``datetime`` type or re-mapping individual
+  columns. See the Database migration notes for how this type is automatically
+  mapped to database types.
```

Comment:
I don't know if these types will be supported by migrations, we might want to figure that out as well.

Reply:
Confirmed timezone type is in phinx postgres config.